examples/interface/package.json

```diff
@@ -12,7 +12,7 @@
     "@starknet-react/core": "^1.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "@bibliothecadao/instaswap-core": "link:instaswap/packages/instaswap-core"
+    "@bibliothecadao/instaswap-core": "link:@bibliothecadao/instaswap-core"
```

Comment:
this won't work will it?

Reply:
this works since I build link under instaswap-core directory but not sdk root path.

I have changed it back to keep everything on the same page.